lib/private/DB/ConnectionFactory.php

```diff
@@ -212,7 +215,7 @@ public function createConnectionParams(string $configPrefix = '', array $additio
 			'tablePrefix' => $connectionParams['tablePrefix']
 		];
 
-		if ($this->config->getValue('mysql.utf8mb4', false)) {
+		if ($this->config->getValue('mysql.utf8mb4', false) && $type==="mysql") {
```

Comment:
```suggestion
		if ($this->config->getValue('mysql.utf8mb4', false) && $type === 'mysql') {
```

To make the linter pass.